.github/workflows/bindings-rust.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           fetch-depth: 0
 
       - name: Install Rust v1.65
-        uses: dtolnay/rust-toolchain@1.65
+        uses: dtolnay/rust-toolchain@stable
         with:
           toolchain: 1.65
```

Comment:
Should we also use the latest rust toolchain instead of `1.65`?

Reply:
From the perspective of Rust SDK, it's fine to use the latest stable version, but I'm not sure if any dependency in the dependency chain is compatible with the latest stable version. Of course, `1.65` is the latest stable version and no issues on this version. In my view, it could be better for us to use the recently published two or three stable versions in our CI workflows. If the latest version caused any compatibility failure, the others can help us.

Reply:
So do we need to bump the version now? Or we can keep the `1.65` for a while.